lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -2237,7 +2283,12 @@ private TaskQuery addOrderCriteria(String columnName, SortDirection sortDirectio
     if (sortDirection == null) {
       sortDirection = SortDirection.ASCENDING;
     }
-    orderBy.add(columnName + " " + sortDirection);
+    orderByInner.add(columnName + " " + sortDirection);
+    if (columnName.startsWith("a") || columnName.startsWith("w") || columnName.startsWith("c")) {
+      orderByOuter.add(columnName.replace(".", "").toUpperCase() + " " + sortDirection);
+    } else {
+      orderByOuter.add(columnName + " " + sortDirection);
+    }
```

Comment:
Please comment this (and why it is necessary). It might be confusing for future devs. Fixing this would overload this ticket. Therefore, a comment is enough for me :)